support.md

```diff
@@ -69,15 +67,15 @@ In the description of your issue, be sure to also include:
     `systemctl status brickman.service -l`.
   * If a screenshot of the EV3 would be helpful, run `fbgrab <some-name>.png`
     on the EV3 and include the picture in your comments.
-<p />
```

Comment:
OK. Make that WTH @dlech. I read `</p>` instead of `<p />`, which does not compute in my brain.